pages/posts/@id/+onBeforeRender.ts

```diff
@@ -6,10 +6,10 @@ import useQueriesState from "../../../stores/queriesState";
 export default async function onBeforeRender(pageContext: PageContextBuiltInServer) {
     let title = "Post Detail";
     const fetchedQueries = useQueriesState.getState().fetchedQueries
-    const { isClientSideNavigation, routeParams: { id } } = pageContext
+    const { routeParams: { id } } = pageContext
 
-    if (isClientSideNavigation === false || fetchedQueries.has(hashKey(postsQueries.detail(id).queryKey).valueOf()) === false) {
-        console.log('posts/id/+onBeforeRender is fetcing... id : ', id)
+    if (fetchedQueries.has(hashKey(postsQueries.detail(id).queryKey).valueOf()) === false) {
```

Comment:
I will change this to `if(!knownQueries.get(queryKey))`, cause we also need to check if the value is false.

[Map & Set](https://javascript.info/map-set) : 
- map.get(key) – returns the value by the key, `undefined` if key doesn’t exist in map.
- map.has(key) – returns true if the key exists, false otherwise.